code/modules/projectiles/guns/specialist.dm

```diff
@@ -822,7 +822,6 @@
 		log_explosion("[key_name(user)] fired a grenade [F] from \a [src] at [AREACOORD(user.loc)].")
 		F.icon_state = initial(F.icon_state) + "_active"
 		F.activate(user)
-		F.updateicon()
```

Comment:
you tested that this is not making any change?

Reply:
tested on both standard and this nade